e2e/relayer/relayer.go

```diff
@@ -15,8 +15,10 @@ const (
 	Rly    = "rly"
 	Hermes = "hermes"
 
-	cosmosRelayerRepository = "damiannolan/rly" //"ghcr.io/cosmos/relayer"
-	cosmosRelayerUser       = "100:1000"        // docker run -it --rm --entrypoint echo ghcr.io/cosmos/relayer "$(id -u):$(id -g)"
+	hermesRelayerRepository = "ghcr.io/informalsystems/hermes"
+	hermesRelayerUser       = "1000:1000"
+	rlyRelayerRepository    = "damiannolan/rly" //"ghcr.io/cosmos/relayer"
+	rlyRelayerUser          = "100:1000"        // docker run -it --rm --entrypoint echo ghcr.io/cosmos/relayer "$(id -u):$(id -g)"
```

Comment:
I think there's a public function for this now, but can't remember

Reply:
looked into this. Using a custom image doesn't use the defaults. Can't use the defaults for `rly` since we're based on Damian's forked image, can't use the default for hermes since `interchaintest` hasn't updated their registry url (still points to docker rather than `ghcr`). I'll probably open a PR for the second case and we can drop both consts for hermes.

Reply:
cool, once we switch off damian's image (probably after v7.1 release) then we can move to the defaults. Is there an issue to track this work? If not, maybe we can open an issue

Reply:
> If not, maybe we can open an issue

yup! done with https://github.com/cosmos/ibc-go/issues/3615

Reply:
re: hermes image/tag updates done https://github.com/strangelove-ventures/interchaintest/pull/571